pytorch_lightning/trainer/connectors/optimizer_connector.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 from pytorch_lightning.utilities import rank_zero_warn
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from timm.scheduler.scheduler import Scheduler as TimmScheduler
```

Comment:
As `timm` is an extra feature, we need to make sure that the code can run even if the user does not have timm installed. Something like this should be done:
https://github.com/PyTorchLightning/pytorch-lightning/blob/0a50bb406fa41dfa6a0e2be52f531a9c81c87d00/pytorch_lightning/loggers/wandb.py#L30-L37